scripts/hydra

```diff
@@ -244,6 +244,21 @@ function update() {
   exit 0
 }
 
+# @cmd Get the logs from containers
+# @arg container_name!        Container name
+# @arg layer!                 Layer name (global-l0, dag-l1, metagraph-l0, currency-l1, data-l1)
+# @option --retroactive_rows   Retroactive rows from logs file (default 10)
```

Comment:
I think we should prefer matching the options of tail as closely as possible for this. What about just `-n` here?

Reply:
Hmm, okay I can do this

Reply:
@AlexBrandes 
https://github.com/Constellation-Labs/euclid-development-environment/blob/78a418c99a111c845e564455aff5be300550eb1f/scripts/hydra#L247C2-L261C1